config.yml

```diff
@@ -95,7 +99,9 @@ production:
 
 staging:
   app:
-    id: staging-aws
+    provisionerId: staging-aws
+    description: |
+      Staging instance for the AWS provisioner.  Not for production use.
```

Comment:
Extra space before "Not"

Reply:
`I     hate you   .` ;)

Reply:
hahah, even in monospace markdown strips consecutive spaces.  See, it's markdown, it doesn't matter.